public/docs/ts/latest/cookbook/production-deploy.jade

```diff
@@ -0,0 +1,327 @@
+include ../_util-fns
+  
+:marked
+  This cookbook describes how to optimize and deploy your Angular app to production.
+
+a#toc
+:marked
+  ## Table of Contents
+  * [Overview](#overview)
+  * [Simplest deploy possible](#simplest-deploy-possible)
+  * [Minimizing your payload](#minimizing-your-payload)
+    * [Fewer files](#fewer-files)
+    * [Smaller files](#smaller-files)
+    * [Template compilation](#template-compilation)
+  * [Angular configuration](#angular-configuration)
+    * [The `base` tag](#the-base-tag)
+    * [enableProdMode](#enableprodmode)
+    * [Lazy loading](#lazy-loading)
+  * [Server configuration](#server-configuration)
+    * [Why fallback to `index.html`?](#why-fallback-to-index-html-)  
+    * [Fallback configuration examples](#fallback-configuration-examples)  
+
+a#overview
+.l-main-section
+:marked
+  ## Overview
+
+  After creating your brand new Angular app you'll want to put it online for the world to see.
+
+  Your development setup is optimized for build speed and rapid iteration, but when
+  deploying to production you'll want to optimize for loading speed and payload size.
+
+  In this chapter you'll see how to deploy your app right now, techniques to reduce your 
+  payload size, plus Angular and server configuration.
+
+.l-main-section
+:marked
+  ## Simplest deploy possible
+
+  The simplest possible way to deploy your app is to use your development environment.
+  It is already working locally after all, so it should work live.
+
+  1. Copy over your local project folder to your server.
+  2. On your server, edit `index.html` to have the correct `<base href="/">` tag. If you are
+  using a subfolder (e.g. `www.mysite.com/myapp/`) it should be `/myapp/` otherwise leave it as `/`.
+  [More on this later](#angular-configuration).
+  3. Configure your server to redirect requests for missing files to `index.html` instead.
+  [More on this later](#server-configuration).
+
+  And this is all you need to publish your app!
+
+  It's not a very good production deploy though. 
```

Comment:
Instead of negative prose, I suggest ... *You've deployed your app, but we can do better for production to optimize the app! Let's learn how.*